Python/Product/PythonTools/PythonTools/Intellisense/BufferParser.cs

```diff
@@ -165,7 +165,8 @@ sealed class BufferParser : IDisposable {
         private void InitBuffer(ITextBuffer buffer, int id = 0) {
             var bi = _services.GetBufferInfo(buffer);
             if (!bi.SetAnalysisEntryId(id)) {
-                Debug.Fail("Buffer is already initialized");
+                // TODO: this can occur when a buffer has multiple views (new window, peek window)
+                //Debug.Fail("Buffer is already initialized");
```

Comment:
FWIW, I'm already working on fixes for this and the TryDispose below, but they're big (need to reverse event hook relationships all over the place...)